src/OrbitBase/ThreadUtilsLinux.cpp

```diff
@@ -51,4 +51,10 @@ void SetCurrentThreadName(const std::string& thread_name) {
   }
 }
 
+pid_t GetCurrentProccessId() {
+  pid_t current_pid = getpid();
+  CHECK(current_pid);
```

Comment:
no need to check - this method always succeeds according to man page.

Reply:
Changed accordingly. Thanks!